app/src/main/java/org/p2p/wallet/swap/jupiter/api/response/quote/JupiterRouteResponse.kt

```diff
@@ -0,0 +1,27 @@
+package org.p2p.wallet.swap.jupiter.api.response.quote
+
+import com.google.gson.annotations.SerializedName
+import org.p2p.wallet.swap.jupiter.api.request.JupiterSwapFeesRequest
+import org.p2p.wallet.swap.jupiter.api.request.SwapRouteRequest
+import org.p2p.wallet.swap.jupiter.repository.model.JupiterSwapMode
+
+data class JupiterRouteResponse(
```

Comment:
maybe we can create just one entity and use it as request and response?

Reply:
really its same

Reply:
yeah, that's my point, so instead of two same entities we can use one?

Reply:
no, make it separate

Reply:
its same in swagger 
https://quote-api.jup.ag/v4/docs/static/index.html